### DISCUSSION TITLE
UI not appearing or UI noninteractive

### DISCUSSION THREAD

**Discussion by MegaTheGoat** [ErinaSugino/Starbound-SBR-Mod#134] [Trouble Shooting] [answered]
Whenever I TP to outpost and click on any sexnode, my characters just stands there in idle pose. Most of the time UI doesn't appear and even when it appears, it's completely noninteractive, leaving my character just standing there on the table.

I don't have any other mod other than  sexbound-reborn and POV Redux 4.2.0.

Here is the log file : 
[starbound.log](https://github.com/user-attachments/files/17193929/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
I'd assume you have downloaded the newest 1.2_r1 release.
The download files for that release have been updated about twice in the last 24h, fixing the issue your log presents. Please re-download the newest version.

**Reply by MegaTheGoat** [reply]
Works like a charm! Thanks.
